console/src/telemetryFilter.ts

```diff
@@ -34,25 +34,25 @@ const dataHandler = path => {
   );
 };
 
-const apiExplorerHandler = () => {
+const apiExplorerHandler = (): string => {
   return API_EXPLORER_PATH;
 };
 
-const remoteSchemasHandler = path => {
+const remoteSchemasHandler = (path: string): string => {
   return (
     REMOTE_SCHEMAS_PATH +
     path.replace(/(\/manage\/)[^/]*(\/\w+.*)$/, '$1REMOTE_SCHEMA_NAME$2')
   );
 };
 
-const eventsHandler = path => {
+const eventsHandler = (path: string): string => {
```

Comment:
As TypeScript can infer that.
```suggestion
const eventsHandler = (path: string) => {
```